Configuration/TCA/Overrides/tt_content_basic_content.php

```diff
@@ -0,0 +1,46 @@
+<?php
```

Comment:
Shouldn't we also add an entry to the New Content Element Wizard as best practise?

```php
ExtensionManagementUtility::addPlugin(
    [
        'label' => 'Example - basic content',
        'description' => 'Content element as demonstration for basic content',
        'group' => 'default',
        'value' => $key,
        'icon' => 'content-text',
    ],
    'CType',
    'examples,
);
```

see also https://github.com/TYPO3-Documentation/Changelog-To-Doc/issues/1012

Reply:
This is not a plugin but a plain content element with some TypoScript. And with https://docs.typo3.org/c/typo3/cms-core/main/en-us/Changelog/13.0/Feature-102834-Auto-registrationOfNewContentElementWizardViaTCA.html they are automatically listed when created  via ExtensionManagementUtility::addTcaSelectItem. Try it out, it is working

Reply:
Yup, that's why "CType" is used (and not plugin)....

But, is description and the icon added? Maybe then also add it to the addTcaSelectItem if this works, I am fine.

Reply:
yes they are added